api/chat.go

```diff
@@ -308,6 +320,49 @@ func (r chatRoutes) handleApprove(ctx tools.TUMLiveContext, msg []byte) {
 	broadcastStream(ctx.Stream.ID, broadcastBytes)
 }
 
+func (r chatRoutes) handleReactTo(ctx tools.TUMLiveContext, msg []byte) {
+	var req wsReactToReq
+	err := json.Unmarshal(msg, &req)
+	if err != nil {
+		log.WithError(err).Warn("could not unmarshal reactTo request")
+		return
+	}
+
+	isAllowed := false
+	for _, reaction := range allowedReactions {
```

Comment:
Early intrusion just because I noticed it: You could use a map for `allowedReactions` and check for 
validity with:
```golang
if _, isAllowed := dict[req.Emoji]; !isAllowed {
   log.Warn("user tried to add illegal reaction")
   return
}
```

Feature looks great btw!